tools/cuttlefish-host-image-installer/metapackage-google/usr/share/cf_docker_co_run.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Step 1. Build Cuttlefish docker image if there is no such one
+cf_image=$(docker image list | grep "cuttlefish-orchestration")
+if [ "$cf_image" == "" ]; then
+  # Build CF docker image
+  cd /home/vsoc-01
+  if [ ! -d "/home/vsoc-01/android-cuttlefish" ]; then
+    echo "git  clone https://github.com/google/android-cuttlefish.git" > android-cuttlefish.log
+    wget https://github.com/google/android-cuttlefish/archive/refs/tags/stable.zip
+    unzip stable.zip
+  fi
+  cd android-cuttlefish-stable/docker/orchestration
+  /bin/bash build.sh &> build.log
+fi
+
+# Step 2. Run CO server
+cd /home/vsoc-01
+if [ ! -d "/home/vsoc-01/cloud-android-orchestration" ]; then
+  echo "git clone https://github.com/google/cloud-android-orchestration.git" > cloud-android-orchestration.log
+  wget https://github.com/google/cloud-android-orchestration/archive/refs/heads/main.zip
```

Comment:
let's have a fixed version of the CO rather than pulling from HEAD everytime, we might need tags on the cloud orchestration repository for it.

Reply:
What is the fixed version or release/tag name you want to use for Cuttlefish Docker flow?

Reply:
created https://github.com/google/cloud-android-orchestration/releases/tag/v0.1.0-alpha

Reply:
Changed script to use fixed version, currently use this release https://github.com/google/cloud-android-orchestration/releases/tag/v0.1.0-alpha